src/app/components/header/header.component.html

```diff
@@ -0,0 +1,32 @@
+<div ngbNav class="nav navbar navbar-expand-lg navbar-dark bg-dark" #nav="ngbNav" id="header">
+    <div class="col col-2 row brand">
+        <label class="burger-icon" (click)="toggle()" id="burger-icon">
+            <div class="burger"></div>
+            <div class="burger"></div>
+            <div class="burger"></div> 
+        </label>
+        <a routerLink="" class="navbar-brand">
+            <img src="../../../assets/img/maya.svg" class="maya">
```

Comment:
Use a variable instead of a static link like this. You can call the variable inside the header.component.ts file.